.gitignore

```diff
@@ -113,3 +113,4 @@ __pycache__/
 #ENV VARIABLES
 .gitignore
 envs
+covhtml/
```

Comment:
Insert a blank line at the end of this file, test_data file and some other file in the tests.

Reply:
I did fix this thanks for the recommendation